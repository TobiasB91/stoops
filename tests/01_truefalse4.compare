Fehler: Fehlende Deklaration von Main
