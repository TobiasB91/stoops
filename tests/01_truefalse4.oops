CLASS TRUE IS | Fehler
END CLASS
