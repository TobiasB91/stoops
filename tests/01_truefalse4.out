Fehler in Zeile 1, Spalte 7: Unerwartetes Symbol TRUE
